src/app/store.js

```diff
@@ -0,0 +1,15 @@
+import { configureStore } from '@reduxjs/toolkit';
```

Comment:
- [ ] Great job so far :+1:  However The structure of the files should follow the `ducks pattern`

eg. Has basic project structure

```
app-root-directory
  |- package.json
  |- README.md
  |- /public
    |- index.html
  |- /src
    |- app.css
    |- index.js
    |- /components
    |- /container
    |- /pages
    |- /redux
  |- /node_modules
```